tests/assets/client.py

```diff
@@ -12,22 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 from pathlib import Path
 
-from playwright.sync_api import Playwright, sync_playwright
+from playwright.sync_api import sync_playwright
 
 
-def main(playwright: Playwright) -> None:
-    for browser_type in [playwright.chromium, playwright.firefox, playwright.webkit]:
-        browser = browser_type.launch()
-        page = browser.new_page()
-        page.goto("https://example.com")
-        here = Path(__file__).parent.resolve()
-        page.screenshot(path=here / f"{browser_type.name}.png")
-        page.close()
-        browser.close()
+def main() -> None:
+    with sync_playwright() as p:
```

Comment:
having additionally a normal test-case in the browser tests for async and sync would be good to have.

Reply:
Okay